src/constants/permissions.ts

```diff
@@ -201,9 +209,10 @@ values.SYSCONSOLE_ANCILLARY_PERMISSIONS = {
     [values.SYSCONSOLE_READ_AUTHENTICATION]: [
         values.READ_JOBS,
     ],
-    [values.SYSCONSOLE_READ_REPORTING]: [
-        values.VIEW_TEAM,
-    ],
+
+    // [values.SYSCONSOLE_READ_REPORTING]: [
+    //     values.VIEW_TEAM,
+    // ],
```

Comment:
Will we just remove this?

Reply:
Yes there is a PR to get rid of all of these on the redux side of things. See https://github.com/mattermost/mattermost-redux/pull/1392 @BenCookie95